backend/mockingbird/src/main/scala/ru/tinkoff/tcb/mockingbird/grpc/GrpcRequestHandler.scala

```diff
@@ -90,9 +90,20 @@ object GrpcRequestHandlerImpl {
 }
 
 object GrpcRequestHandler {
+  object keys {
+    val correlationId = io.grpc.Metadata.Key.of("X-Correlation-ID", io.grpc.Metadata.ASCII_STRING_MARSHALLER)
```

Comment:
I think these fields should be configurable as well

Reply:
OK, I will add separate config for it.

Reply:
Oh, it's from previous changes. It is not actual for now. I misunderstood you. Now HTTP and gRPC have the same configuration, see file `backend/mockingbird/src/main/scala/ru/tinkoff/tcb/mockingbird/api/Tracing.scala` line 20. So, I will delete object `keys`, because it is unused.

Reply:
Removed, tracing for gRPC and HTTP have the same configuration. If you think that adding the separate config for extracting/injecting gRPC I will do it.